### ISSUE TITLE
Meta Tracker for Broken Spawn

### ISSUE THREAD

**Issue by jeremybris** [RPCS3/rpcs3#7522]
## Quick summary

- The player spawns out of bounds. 
- The player falls through the floor.

## Details

- [ ] #7492 Resistance: Fall of Man [NPEA00430]
- [ ] #7130 Ratchet & Clank: Tools of Destruction [BCES00052]
- [x] #4258 Ratchet & Clank: Tools of Destruction [BCES00052]
- [x] [#5806](https://github.com/RPCS3/rpcs3/issues/5806) God Of War 3 [BCUS98111]
- [x] Skate 2 [BLES00461] (Respawning repeatedly randomly triggers a white screen)

## Discussion
Does anyone disagree and think one or more bugs are unrelated? 
Did I miss a bug report related to this?
I hope this can be helpful.
If this is unhelpful, I will close this report.

**Comment by jeremybris** [comment]
Interestingly, this was just fixed for me in Ratchet & Clank: Tools of Destruction [BCES00052] in #7304 !!! Can anyone confirm or deny the others were fixed? I will do more testing.

**Comment by jeremybris** [comment]
I think this fixed
- Ratchet & Clank: Tools of Destruction [BCES00052]
- Skate 2 [BLES00461]

Not fixed. It is an unrelated bug.

- Sly 2 [BCES00968]


The others I can not comment on. I have not experienced them.

**Comment by Cervoxx** [comment]
- Ratchet & Clank: Tools of Destruction [BCUS98127]
Not fixed, but significantly reduced. Tested this by closing and reopening the game multiple times, and it happened only once. 

Logs: https://files.catbox.moe/5s0455.gz

**Comment by lainiwaku** [comment]
> 
> 
> Interestingly, this was just fixed for me in Ratchet & Clank: Tools of Destruction [BCES00052] in #7304 !!! Can anyone confirm or deny the others were fixed? I will do more testing.

on resistance fall of man it's a regression
before #7304 = player spawn and jump out of the map 
after #7304 =  black screen

**Comment by lainiwaku** [comment]
[RPCS3ASMJIT.log.gz](https://github.com/RPCS3/rpcs3/files/4223667/RPCS3ASMJIT.log.gz)
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/4223682/RPCS3.log)

**Comment by digitaldude555** [comment]
Either #7492 #7130 #5806 needs closing or this issue needs closing. @AniLeo  what you think.

**Comment by jeremybris** [comment]
> * Ratchet & Clank: Tools of Destruction [BCUS98127]
> 
> Not fixed, but significantly reduced. Tested this by closing and reopening the game multiple times, and it happened only once.
> 
> Logs: https://files.catbox.moe/5s0455.gz

@Cervoxx I'm glad someone else is testing the games. Lainiwaku says he found a workaround by doing  "SPU DECODER : Interpreter (fast)", can you try that, and tell us if it fixes the bug, please?

**Comment by Cervoxx** [comment]
> @Cervoxx I'm glad you tested it. Lainiwaku says he found a workaround by doing "SPU DECODER : Interpreter (fast)". Will you try that, and tell me if it fixes the bug, please?

Will do soon. Not sure if its necessary or not, but bear in mind that my Region ID is different to the one you have listed. As far as I understand I've got the american version whilst you've got the european version, assuming I'm reading this correctly. 

![image](https://user-images.githubusercontent.com/12489208/74867509-68291780-531a-11ea-9f27-d4e0cdb4893d.png)

**Comment by elad335** [comment]
Test regressions with #7532

**Comment by jeremybris** [comment]
Here is the link to #7532 https://ci.appveyor.com/api/buildjobs/pe9poxs8twn8nig3/artifacts/rpcs3-v0.0.8-6fb3307a_win64.7z

**Comment by Cervoxx** [comment]
> Test regressions with #7532

You will need to make this build if you want me to test it. A windows build wont work for me since I use linux so I need an appimage, and your documentation on manually building rpsc3 for linux is outdated. And as a user unfamiliar with git, I gave up even after setting up a ubuntu 18.04 VM in an attempt to build that branch where the pull request originated from. A bit too much frustration here.

> @Cervoxx I'm glad you tested it. Lainiwaku says he found a workaround by doing "SPU DECODER : Interpreter (fast)". Will you try that, and tell me if it fixes the bug, please?

Still on my radar to do this, pardon my delay.

**Comment by Yahfz** [comment]
Add Grand Theft Auto IV NPEB00882

**Comment by jeremybris** [comment]
> > Test regressions with #7532
> 
> @elad335
> You will need to make this build if you want me to test it. A windows build wont work for me since I use linux so I need an appimage, and your documentation on manually building rpsc3 for linux is outdated. And as a user unfamiliar with git, I gave up even after setting up a ubuntu 18.04 VM in an attempt to build that branch where the pull request originated from. A bit too much frustration here.
> 
> @jeremybris
> 
> > @Cervoxx I'm glad you tested it. Lainiwaku says he found a workaround by doing "SPU DECODER : Interpreter (fast)". Will you try that, and tell me if it fixes the bug, please?
> 
> Still on my radar to do this, pardon my delay.

No need to try that anymore, thanks anyways. Another person confirmed the new build works for Ratchet and Clank. It should work for you too when publicly posted.

**Comment by dio-gh** [comment]
Is it not that GTA IV just requires accurate xfloat? @Yahfz

**Comment by jeremybris** [comment]
**I will be closing this soon. In conlusion, thanks to elad's work,**

- #7492 - Fixed. Should be closed.
- #7130 - Fixed. Should be closed.
- #4258 - Partially fixed. Should either be updated, or closed and a new issue could be opened, because the majority of issues have been fixed.
- #5806 - Probably fixed, the thread is dead, or both. Should probably be closed.
- Skate 2 [BLES00461] - Fixed

**Unrelated Bugs**
- #5289 Sly 2 [BCES00968]
- Grand Theft Auto IV [NPEB00882] - Please try accurate xfloat, or open a new issue.

**Comment by digitaldude555** [comment]
@jeremybris uncheck all of them except skate they are not fixed and pr was closed.

**Comment by web1018** [comment]
Yes. I can confirm that it's still not fixed. Elad closed the PR and the one that he has merged instead does not fix the regression. Resistance: Fall of Man is back to having black screen after death and the issue is still standing.

**Comment by jeremybris** [comment]
> Yes. I can confirm that it's still not fixed. Elad closed the PR and the one that he has merged instead does not fix the regression. Resistance: Fall of Man is back to having black screen after the player dies and the issue is still standing.

Oh, ok. I thought he closed that pull because he included the fix in his next pull. I'll leave this open, until it is fixed in the master.

**Comment by web1018** [comment]
> > Yes. I can confirm that it's still not fixed. Elad closed the PR and the one that he has merged instead does not fix the regression. Resistance: Fall of Man is back to having black screen after the player dies and the issue is still standing.
> 
> Ohh understood, thanks for correcting me. I thought elad closed that PR because elad included the fix in his next PR. I'll leave this open, until it is fixed in the master.

I thought so myself after reading his comments. But when I tested it, I saw that it was not fixed.

**Comment by lainiwaku** [comment]
maybe @elad335 was thinking  his PR  would fix it so he closed #7532   ?

**Comment by digitaldude555** [comment]
Gonna need an update on whats fixed now that #7565 merged.

**Comment by digitaldude555** [comment]
@jeremybris resistance is fixed you can close this issue now.

**Comment by AniLeo** [comment]
Closing as mentioned issues should now be fixed

**Comment by jeremybris** [comment]
> Gonna need an update on whats fixed now that #7565 merged.

The reports are mixed. It seems like the "spawning out of bounds" bug has been replaced by "the emulator crashes when you die", which may have been fixed https://github.com/RPCS3/rpcs3/issues/7562#issuecomment-590516698 or randomly it happens randomly https://github.com/RPCS3/rpcs3/pull/7565#issuecomment-590080172.

- #7492 - https://github.com/RPCS3/rpcs3/pull/7565#issuecomment-59006539 Resistance: Fall of Man [NPEA00430]
- #7130 - https://github.com/RPCS3/rpcs3/pull/7565#issuecomment-592896362 Ratchet & Clank: Tools of Destruction [BCES00052] I can confirm that it is fixed for me, but it was fixed for me before the 7565 PR and Cervoxx, said he/she could recreate "closing and reopening the game multiple times".
- #4258 - Partially fixed. Should be updated.
- #5806 - https://github.com/RPCS3/rpcs3/issues/5806#issuecomment-589243608 Fixed and closed.
- Skate 2 [BLES00461] - Fixed for me

In my opinion, this issue should be closed, and a new issue should be opened. It may be an unrelated bug or nonexistent.
